addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
It's explicitly called from within `run` https://github.com/asakusuma/ember-spaniel/blob/master/addon/spaniel-engines/ember-spaniel-engine.js#L19

So it's guaranteed to not be undefined.